src/lib/styles/themes/lightTheme/lightTheme.module.scss

```diff
@@ -5,11 +5,20 @@ $lightTheme: (
   brandSecondary: rgb(199, 0, 255),
   brandTertiary: rgb(218, 13, 255),
   
-  primaryFontFamily: $lightTheme-primary-font-family-bold,
-  secondaryFontFamily: $lightTheme-secondary-font-family,
-  tertiaryFontFamily: $lightTheme-tertiary-font-family,
-  
-  gradientFontFamily: $lightTheme-gradient-title-font-family,
+  // Old font styles that we should be replacing and then deleting. >
+  // primaryFontFamily: $lightTheme-primary-font-family-bold,
+  // secondaryFontFamily: $lightTheme-secondary-font-family,
+  // tertiaryFontFamily: $lightTheme-tertiary-font-family,
+  // gradientFontFamily: $lightTheme-gradient-title-font-family,
+
+  // New font styles I'm adding right now. >
+  featureFontFamily: $lightTheme-feature-font-family,
+  primaryFontFamilySemiBold: $lightTheme-primary-font-semibold,
+  primaryFontFamilySemiBoldItalic: $lightTheme-primary-font-semibold-italic,
+  primaryFontFamilyBoldItalic: $lightTheme-primary-font-bold-italic,
+  primaryFontFamilyRegular: $lightTheme-primary-font-regular,
+  primaryFontFamilyExtraBoldItalic: $lightTheme-primary-font-extrabold-italic,
+  bodyFontFamily: $lightTheme-body-font-family,
```

Comment:
Probably should remove above comments (but maybe good for context)

Reply:
I'm gonna remove this because I've been able to run it with no conflicts so there are no references to these old fonts I can see.